app/models/study.rb

```diff
@@ -1039,6 +1060,26 @@ def default_annotation_type
     end
   end
 
+  # helper to return default annotation name
+  def default_annotation_name
+    if self.default_options[:annotation].nil?
```

Comment:
We might want to use `.blank?` here instead of `.nil?`.  I know that existing code calls `.nil?` but I think `.blank?` is safer.